docs/vtex-io/Getting-Started/getting-started-3/vtex-io-documentation-2-prerequesites.md

```diff
@@ -24,14 +24,18 @@ The [Edition app](https://developers.vtex.com/docs/guides/vtex-io-documentation-
 vtex edition get
 ```
 
-If the current edition differs from `vtex.edition-store@3.x`, [open a support ticket](https://help.vtex.com/support?/cultureInfo=en-us) and request the installation of the `vtex.edition-store@3.x` app in your VTEX account.
+If the current edition is older than `vtex.edition-store@3.x`, [open a support ticket](https://help.vtex.com/support?/cultureInfo=en-us) and request the installation of the `vtex.edition-store@3.x` or a newer one in your VTEX account.
 
-> ⚠️ By default, every VTEX store comes with the Edition app configured for building storefronts with [Legacy CMS Portal](https://help.vtex.com/tutorial/what-is-cms--EmO8u2WBj2W4MUQCS8262) installed. Therefore, requesting the installation of the `vtex.edition-store@3.x` Edition app is necessary for both new VTEX accounts and accounts previously using the [Legacy CMS Portal](https://help.vtex.com/tutorial/what-is-cms--EmO8u2WBj2W4MUQCS8262).
+>ℹ Check out our  [Edition App documentation](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app) to know more about its different versions and install the correct one.
+
+> ⚠️ By default, every VTEX store comes with the Edition app configured for building storefronts with [Legacy CMS Portal](https://help.vtex.com/tutorial/what-is-cms--EmO8u2WBj2W4MUQCS8262) installed. Therefore, requesting the installation of the `vtex.edition-store@3.x` or newer Edition app is necessary for both new VTEX accounts and accounts previously using the [Legacy CMS Portal](https://help.vtex.com/tutorial/what-is-cms--EmO8u2WBj2W4MUQCS8262).
 
 ### Step 2 - Implementing the VTEX Intelligent Search
 
 Before you can begin building your storefront with Store Framework, you need to configure [Intelligent Search](https://help.vtex.com/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb).
 
+>ℹ️ If you use the `vtex.edition-store@3.x` version, you must also install the Search apps. If you use the `vtex.edition-store@5.x`, you must start the Search Integration process directly, as the Intelligent Search is already configured by default in this version.
```

Comment:
Aqui você pode usar anchor links (e.g., `[Installing the Search apps](#installing-the-search-apps)`) para seções específicas da documentação